package.json

```diff
@@ -6,21 +6,31 @@
   "devDependencies": {
     "@commitlint/config-conventional": "^19.4.1",
     "@fingerprintjs/eslint-config-dx-team": "^0.1.0",
+    "@fingerprintjs/fingerprintjs-pro-server-api": "^5.0.0",
```

Comment:
Let's discuss this. It is not ideal to make proxy integration dependent on this SDK. As far as I can see, we only use it to import `EventResponse` type. I'd suggest to use `unknown` type, or copy-paste current `EventResponse` type, or something better, rather than depending on the Node SDK. Pls let me know your thoughts @Orkuncakilkaya @TheUnderScorer @ilfa @JuroUhlar

Reply:
I believe copy-pasting the current EventResponse type is not a viable solution because it will eventually change. 
Using the unknown could help, but the customer must be careful about the actual type before casting it to something else. If we could have types such as EventResponse separated from the SDK itself, it could help a lot, but it's not something that should be discussed here, IMHO.

Reply:
Since we only import the type, we could probably move it to dev dependency

Reply:
It's already a dev dependency, did you meant something else?

Reply:
I missed that it was `devDependency`. In this case we can keep it as-is. Thanks!